spec/spec_helper.rb

```diff
@@ -10,5 +28,14 @@
 
   config.expect_with :rspec do |c|
     c.syntax = :expect
+    c.include_chain_clauses_in_custom_matcher_descriptions = true
   end
+
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = true
+  end
+  config.shared_context_metadata_behavior = :apply_to_host_groups
 end
+
+RSpec::Expectations.configuration.on_potential_false_positives = :nothing
```

Comment:
Rspecs configurations have warning messages for testings raised errors safely.So to suppress those warnings I add this here.But we can remove it.It will not impact anything.

Reply:
I was looking for a deeper explanation on why putting these settings would suppress warnings and why warnings are important. The better way is to fix the root causes that led to those warnings instead of suppressing them.